extensions/open-remote-ssh/src/serverConfig.ts

```diff
@@ -35,6 +35,7 @@ export async function getVSCodeServerConfig(): Promise<IServerConfig> {
 	const productJson = await getVSCodeProductJson();
 
 	const customServerBinaryName = vscode.workspace.getConfiguration('remoteSSH.experimental').get<string>('serverBinaryName', '');
+	const customDataFolderName = vscode.workspace.getConfiguration('remoteSSH').get<string>('serverInstallPath', '');
```

Comment:
SUPER SUPER nit, could set default here rather than later

```suggestion
	const serverDataFolderName = vscode.workspace.getConfiguration('remoteSSH').get<string>('serverInstallPath', productJson.serverDataFolderName);
```